docs/docs/UserGuide/run-local.md

```diff
@@ -0,0 +1,56 @@
+
+
+## **Run as a local Java/Scala program**
+You can try BigDL program, e.g., the [lenet](https://github.com/intel-analytics/BigDL/tree/master/spark/dl/src/main/scala/com/intel/analytics/bigdl/example/lenetLocal) training, testing and prediction as a local Java/Scala program. 
+
+To run the BigDL model as a local Java/Scala program, user needs to set Java property `bigdl.localMode` to `true`. If user wants to specify how many cores to be used for training/testing/prediction, he needs to set Java property `bigdl.coreNumber` to the core number. User can either call `System.setProperty("bigdl.localMode", "true")` and `System.setProperty("bigdl.coreNumber", core_number)` in the Java/Scala code, or pass -Dbigdl.localMode=true and -Dbigdl.coreNumber=core_number when runing the program.
+
```

Comment:
For example, you may run the LetNet model as a local Scala/Java program as follows: